### PR TITLE
Add minio service definition in docker compose and update the environ…

### PR DESCRIPTION
**Pull request by minaxolone**
…ment accordingly

### CODE REVIEW

File: api/.env.example [hunk 1]
```diff
@@ -1,9 +1,21 @@
-S3_REGION="fr-par"
-S3_ENDPOINT="https://s3.fr-par.scw.cloud"
-S3_ACCESS_KEY_ID="SCW43G0RMHJ2PJXNH72G"
-S3_SECRET_ACCESS_KEY="************************************"
+#Use this for S3 as object store
```
Comment (minaxolone) [inline api/.env.example:1]: you can remove those comments

File: ol-fyi-local-infra/compose.yml [hunk 2]
```diff
@@ -40,6 +40,17 @@ services:
       - 6379:6379
       # - 8001:8001
 
+  minio:
+    image: minio/minio
```
Comment (minaxolone) [inline ol-fyi-local-infra/compose.yml:44]: specify the version [RELEASE.2024-05-10T01-41-38Z](https://hub.docker.com/layers/minio/minio/RELEASE.2024-05-10T01-41-38Z/images/sha256-ea0203747a6b779d26ceee879ff9c1d8b70c0d10196a4f969f8ceb4d1e3904bb?context=explore) for instance.

File: ol-fyi-local-infra/compose.yml [hunk 3]
```diff
@@ -40,6 +40,17 @@ services:
...
+    command: server /data
+    environment:
+      MINIO_ROOT_USER: minioadmin
+      MINIO_ROOT_PASSWORD: minioadmin123
+    volumes:
+      - /mnt/data:/data
```
Comment (minaxolone) [inline ol-fyi-local-infra/compose.yml:50]: replace with `- ./volumes/minio/data:/data`

File: api/.env.example (hunk 1 above)
Comment (saki-osive) [reply api/.env.example]: Acknowledged and pushed

File: ol-fyi-local-infra/compose.yml (hunk 2 above)
Comment (saki-osive) [reply ol-fyi-local-infra/compose.yml]: Acknowledged and pushed

File: ol-fyi-local-infra/compose.yml (hunk 3 above)
Comment (saki-osive) [reply ol-fyi-local-infra/compose.yml]: Acknowledged and pushed

**Pull request merged**